features/FEATURE_BLE/ble/pal/PalGap.h

```diff
@@ -649,6 +649,31 @@ struct Gap {
         disconnection_reason_t disconnection_reason
     ) = 0;
 
+    /** Enable or disable privacy feature
+     * @param enable whether to enable privacy feature
+     *  
+     * @return BLE_ERROR_NONE if the request has been successfully sent or the
+     * appropriate error otherwise.
+     * 
+     * @note: See Bluetooth 5 Vol 3 Part C: 10.7 Privacy feature.
+     */ 
+    virtual ble_error_t set_privacy(
```

Comment:
I'm not sure we need that one as functions and events in the pal are privacy ready;  however a function that get if the controller supports privacy would be useful.